Content.Client/UserInterface/Controls/ListContainer.cs

```diff
@@ -212,6 +214,10 @@ protected override Vector2 ArrangeOverride(Vector2 finalSize)
          * var end = (int) 2;
          *
          */
+
+        if (Data.Count == 0 || _itemHeight <= 0)
+            return finalSize;
```

Comment:
Does this work if the ListContainer has items, then has 0 items?

Reply:
It seemed to still work fine? I'm not really sure if it was necessary, I really just added this to make sure a divide by zero doesn't somehow happen while calculating `_topIndex`, but it seems to work fine after your changes so I'll close this pr